app/Controllers/CsvImportController.php

```
@@ -1326,6 +1319,26 @@ classificazione_dewey = ?,
 
         $stmt->execute();
         $stmt->close();
+
+        $this->syncImportedSeries($db, $bookId, $collana, $numeroSerie);
+    }
+
+    private function syncImportedSeries(\mysqli $db, int $bookId, ?string $collana, ?string $numeroSerie): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)